lua/telescope/config.lua

```diff
@@ -279,6 +290,33 @@ local telescope_defaults = {
   ]],
   },
 
+  cache_picker = {
+    {
+      num_pickers = 20,
```

Comment:
I really think this should be 1 by default. If users are interested in storing lots of em, they can do that.

Otherwise, this lets them do the most common use case, which is resuming the last thing they did. I don't think this will happen often to store a bunch of them.